public/js/update-experimenter.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+function submitUpdate() {
+    var name = $('#name').val();
+    var username = $('#username').val();
```

Comment:
I've been thinking about letting users change their username, and I think at least for now it's best not to allow this. It requires us to propagate the change onto microworlds and runs, and it should be a very uncommon thing to do anyway. So let's please take out the option to do so through this form.